app/components/jobseekers/organisation_overviews/schools_component/schools_component.html.slim

```diff
@@ -73,6 +73,6 @@ section#school-overview class="govuk-!-margin-bottom-5"
   - if vacancy.organisations.any?(&:geopoint)
     section#school-location
       h3.govuk-heading-l.section-heading = t("schools.school_location.other")
-
-      div id="map" role="presentation" aria-hidden="true" aria-label=t("schools.map_aria_label") data-schools=organisation_map_data
-      script defer=true src="https://maps.googleapis.com/maps/api/js?key=#{GOOGLE_MAPS_API_KEY}&callback=initMap"
+      = render(MapComponent.new(items: [{type: 'vacancy', params: { type: 'organisation', id: vacancy.id }}].to_json, zoom: 11))
```

Comment:
Perhaps 'school group' is better than organisation? 🤷‍♂️

Reply:
Wait, the type in `app/components/jobseekers/organisation_overviews/school_component/school_component.html.slim` and `app/components/jobseekers/organisation_overviews/school_group_component/school_group_component.html.slim` is 'school'  but

in `app/components/jobseekers/organisation_overviews/schools_component/schools_component.html.slim` is organisation?

Reply:
yes you can get a marker for a vacancy's school or a bunch of markers for a vacancy's organisations schools if its at multiple location. the data is retrieved in a slightly different way. @cesidio  is going the give the api side some love as my api controller is a bit ugly looking and i am sure he might change some of the api terminolgy to something a bit more sensible whilst he is at it